pkg/controller/factory/controller_context.go

```diff
@@ -107,3 +111,17 @@ func namespaceChecker(interestingNamespaces []string) func(obj interface{}) bool
 		return false
 	}
 }
+
+// rateLimitedAddAfterWorkqueue is like workqueue.RateLimitingInterface, but takes the
+// rate limiter into account on AddAfter.
+type rateLimitedAddAfterWorkqueue struct {
+	workqueue.RateLimitingInterface
+	rateLimiter workqueue.RateLimiter
+}
+
+func (q rateLimitedAddAfterWorkqueue) AddAfter(item interface{}, duration time.Duration) {
```

Comment:
nit: would explain what this does comparing to default in godoc

Reply:
done